controllers/update_pods_test.go

```diff
@@ -208,6 +208,92 @@ var _ = Describe("update_pods", func() {
 		})
 	})
 
+	Context("Validating getFaultDomainsWithUnavailablePods", func() {
+		var cluster *fdbv1beta2.FoundationDBCluster
+		var processGroupsWithFaultDomains map[fdbv1beta2.FaultDomain]fdbv1beta2.None
+
+		BeforeEach(func() {
+			cluster = internal.CreateDefaultCluster()
+			Expect(k8sClient.Create(context.TODO(), cluster)).NotTo(HaveOccurred())
+			result, err := reconcileCluster(cluster)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(result.Requeue).To(BeFalse())
+			Expect(k8sClient.Get(context.TODO(), ctrlClient.ObjectKeyFromObject(cluster), cluster)).NotTo(HaveOccurred())
+		})
+
+		JustBeforeEach(func() {
+			processGroupsWithFaultDomains = getFaultDomainsWithUnavailablePods(context.Background(), clusterReconciler, cluster)
+		})
+
+		When("A Process Group has a Pod with pending condition", func() {
```

Comment:
```suggestion
		When("a Process Group has a Pod with pending condition", func() {
```

Those `When` and `Context` statements are put together, having a capital letter in the mid of a sentence looks weird :)